footer.php

```diff
@@ -9,7 +9,12 @@
                     <a href="<?php echo SITE_URL ?>submissions.php">Submissions</a>,
                     <a href="<?php echo SITE_URL ?>apply-submission.php">Apply Submission</a>,
                     <a href="<?php echo SITE_URL ?>view-submission.php">View Submission</a>,
-                    <a href="<?php echo SITE_URL ?>actions-job.php">Create-Edit Job</a>,
+                    <?php error_reporting(0);
```

Comment:
yeee, that's a bad idea. We need the error reporting.
```suggestion
                    <?php
```

Reply:
It's always better to just fix the error instead of hiding it. The error will still be there, just not visible.